setup.py

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
 from pydicom import __version__
-
+from glob import glob
+import os
```

Comment:
oh cool! I thought there might be some logic there - I've been doing the reverse I think - adding modules like sys/os at the bottom, and the rest are abc order. I just updated the PR to conform to this, thanks for the pointer.